src/main/java/com/ioet/bpm/skills/config/SwaggerConfig.java

```diff
@@ -25,7 +25,7 @@ public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .select()
                 .apis(RequestHandlerSelectors.any())
-                .paths(regex("/skills.*"))
+                .paths(regex("/*.*"))
```

Comment:
Why did you change that line?
If we provide a regex like before, that will prevent to render the basic-error-controller in swagger-ui.

Reply:
I should not have committed this change. Now it is removed. Thank you